documentation/docs/validator-zone/external-chains/fantom.md

```diff
@@ -0,0 +1,103 @@
+# Set up your Fantom Opera node
+-----------
+## Disclaimer
+!> :fire: Please note that the guide below may not always be up to date with the latest version or updates from the project, so make sure you always check the official documentation first and run the latest version of software and required dependencies. If you find a problem, please submit an issue to this repository following the template.
+
+
+## Prerequisites
+- Complete all steps from [Setup with Docker](/setup/setup-with-docker.md) or [Setup with Binaries](/setup/setup-with-binaries.md)
+- Minimum hardware requirements: 4 vCPU+, 100GB+ free storage space.
+- MacOS or Ubuntu 18.04+
+- Build-essential packages
+- Golang
+- [Official Documentation](https://docs.fantom.foundation/staking/run-a-read-only-node)
+
+
+## Install required dependencies
+In order to build the `go-opera`, you first need to install all of the required dependencies.
+
+##### 1. Update and install `build-essential`
+
+```bash
+sudo apt-get update
+sudo apt-get -y upgrade
+sudo apt-get install -y build-essential
+```
+
+##### 2. Install `golang`
+
+```bash
+wget https://dl.google.com/go/go1.17.5.linux-amd64.tar.gz
+sudo tar -xvf go1.17.5.linux-amd64.tar.gz -C /usr/local/
+export GOROOT=/usr/local/go
+export GOPATH=$HOME/go
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+```
+
+## Install Go-Opera
+
+##### 1. Checkout and build go-opera
+Use the commands below to clone the `go-opera` repository, checkout correct version and compile the binary.
+
+```bash
+git clone https://github.com/Fantom-foundation/go-opera.git
+cd go-opera/
+git checkout release/1.0.2-rc.5
+make
+```
+
+##### 2. Download the genesis file
+
+```bash
+cd build/
+wget https://opera.fantom.network/testnet.g
+```
+
+##### 3. Start the Fantom Opera node
+
+In this guide we are using `tmux` to run the `opera-go` process in the background. In case you don't have tmux installed, then you can do so with:
+
+```bash
+apt install tmux
+```
+
+Now create a new session called `fantom`:
+
+```bash
+tmux new -s fantom
+```
+
+Once you're inside the newly created tmux session, start the `opera-go`:
```

Comment:
We can use a service  file maybe ? Usually helps with open file limit

Reply:
> We can use a service file maybe ? Usually helps with open file limit

I've used service files on other guides except this one, to diversify a bit but essentially it's down to an user to use his preferred method. I haven't encountered "too many open files" ERR on EVM chains yet but can also add a note.